docs/changes.rst

```diff
@@ -77,6 +77,15 @@ Enhancements
   This does not change the behavior, but increases code clarity.
   Clarified that in the documentation of  `CIMDateTime`. See issue #698.
 
+* Extend cim_http.py, cim_operations.py, _statistics.py to handle optional
```

Comment:
In the change history, I don't think we should talk about our internal modules. Describe it from a user perspective, instead.